lib/index.js

```diff
@@ -15,5 +36,7 @@ exports.parseForESLint = function(code, options) {
 };
 
 exports.parseNoPatch = function(code, options) {
+  options = options || {};
```

Comment:
This looks like an old API that is no longer used. Since we're making a breaking change here anyway, I made another PR to remove this method: https://github.com/babel/babel-eslint/pull/733

The changes here are just a stopgap to allow tests to pass in the interim.